src/firebase.js

```diff
@@ -2,16 +2,25 @@ import firebase from "firebase";
 
 // Your web app's Firebase configuration
 // For Firebase JS SDK v7.20.0 and later, measurementId is optional
+// const firebaseConfig = {
+//   apiKey: "AIzaSyCKlbiyPeccZc_E5wnfYs_wApRw3gjKENY",
+//   authDomain: "mainereentry.firebaseapp.com",
+//   projectId: "mainereentry",
+//   storageBucket: "mainereentry.appspot.com",
+//   messagingSenderId: "844001971791",
+//   appId: "1:844001971791:web:85349a067d13a6b32e643a",
+//   measurementId: "G-P1RQH1BG36",
+// };
+
 const firebaseConfig = {
-  apiKey: "AIzaSyCKlbiyPeccZc_E5wnfYs_wApRw3gjKENY",
-  authDomain: "mainereentry.firebaseapp.com",
-  projectId: "mainereentry",
-  storageBucket: "mainereentry.appspot.com",
-  messagingSenderId: "844001971791",
-  appId: "1:844001971791:web:85349a067d13a6b32e643a",
-  measurementId: "G-P1RQH1BG36",
```

Comment:
"I kindly request that you revert the recent changes made to the `'firebase'` config, in order to allow for proper testing to be conducted. Your attention to this matter would be greatly appreciated."